lib/howl/keymap.moon

```diff
@@ -16,7 +16,9 @@
     tab:              'editor-smart-tab'
     shift_tab:        'editor-smart-back-tab'
     backspace:        'editor-delete-back'
+    ctrl_backspace:   'editor-delete-back-word'
```

Comment:
OK, Option+* variants will work too. I think Word used Command+Backspace, Atom uses Option+Backspace and Lifehacker says both work:

http://lifehacker.com/5803041/the-mac-os-x-delete-key-it-goes-both-ways

Perhaps just add commented out code using `option_backspace` etc. in the osx section then? Next time I fire up Howl on OSX I'll try and set up the the `option` key translation.